### ISSUE TITLE
Is this project still maintained?

### ISSUE THREAD

**Issue by floviolleau** [hanneskod/libmergepdf#83]
Hi,

Thanks for this project. Open source community really appreciate 😊

I have one question : I don't see any update since 4y. Are we good security wize?

Thanks

**Comment by williamdes** [comment]
I guess #75 is a good sign ?

**Comment by FabianKoestring** [comment]
I dont think so. PHP 8.5 is releases and there is still https://github.com/hanneskod/libmergepdf/pull/77.

**Comment by kaystrobach** [comment]
merged right now ... 😄

**Comment by kaystrobach** [comment]
I have merge rights, need to check wether i can do more. we use the project with 8.x so yes we maintain it for us in the company.
